### DISCUSSION TITLE
Is there a plan to have a .NET 8-compiled version of Prism.WPF / Prism.DryIoc?

### DISCUSSION THREAD

**Discussion by david-au** [PrismLibrary/Prism#3040] [WPF]
Hi, with all the performance improvements done in .NET 8, is there a plan to have a .NET 8-compiled version of Prism.WPF / Prism.DryIoc some time in 2024? Thanks.

**Comment by dansiegel** [comment]
It's something we can look at now that it's the current LTS version... however even using the net6 compiled binary if your application is using net8 you'll already get the performance benefits.
